integration_test/issue_wizard/ivido/ivido_demo_client.dart

```diff
@@ -0,0 +1,190 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+import 'package:irmamobile/src/models/session.dart';
+import 'package:irmamobile/src/screens/issue_wizard/widgets/issue_wizard_success_screen.dart';
+import 'package:irmamobile/src/screens/issue_wizard/widgets/wizard_scaffold.dart';
+import 'package:irmamobile/src/screens/scanner/widgets/qr_scanner.dart';
+import 'package:irmamobile/src/widgets/collapsible.dart';
+import 'package:irmamobile/src/widgets/irma_card.dart';
+import 'package:irmamobile/src/widgets/irma_markdown.dart';
+import 'package:irmamobile/src/widgets/irma_stepper.dart';
+import 'package:irmamobile/src/widgets/issuer_verifier_header.dart';
+import 'package:irmamobile/src/widgets/session_progress_indicator.dart';
+
+import '../../helpers/helpers.dart';
+import '../../helpers/issuance_helpers.dart';
+import '../../irma_binding.dart';
+import '../../util.dart';
+
+Future<void> ividoDemoClientTest(WidgetTester tester, IntegrationTestIrmaBinding irmaBinding) async {
+  await pumpAndUnlockApp(tester, irmaBinding.repository);
+
+  // Navigate to the scanner screen
+  await tester.tapAndSettle(find.byKey(const Key('nav_button_scanner')));
```

Comment:
In Firebase I once saw the issue that we cannot give camera permissions on iOS. This caused a permission notification to be present all the time. Is this still the case?

Reply:
In the test the permission dialog is shown but the test continues to run anyway. 

https://console.firebase.google.com/project/yivimobile/testlab/histories/bh.d812b7fd3ed364b4/matrices/6047455985802105590/executions/bs.eeabcecff89e561a/videos (12:00)

Do you think that this is an issue?

Reply:
As long as the test is at the end, it's maybe fine. Previously, I disabled a test on iOS that had this same issue. https://github.com/privacybydesign/irmamobile/blob/ca5b7cf0ad5de830782aa164a39426c8a5cc271d/integration_test/home_test.dart#L94

That test was somewhere at the beginning, so then all tests would get the permission overlay.